wiki/Tournaments/CJB/2018/4K/en.md

```diff
@@ -0,0 +1,145 @@
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 1 month of osu!supporter |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 15 CNY |
+
+## Organization
+
+| Position | Members |
+| :-: | :-- |
+| Organizers | ![][flag_CN] [hans1999](https://osu.ppy.sh/users/6679329), ![][flag_CN] [Flow Light](https://osu.ppy.sh/users/6705256) |
+| Manager | ![][flag_CN] [SSV Normandy](https://osu.ppy.sh/users/6475977) |
+| Map Selectors | ![][flag_CN] [Tofu1222](https://osu.ppy.sh/users/6089608), ![][flag_CN] [hans1999](https://osu.ppy.sh/users/6679329) |
+| Streaming Judgements | ![][flag_CN] [aiyulu](https://osu.ppy.sh/users/189617), ![][flag_CN] [Rasis](https://osu.ppy.sh/users/1209363), ![][flag_CN] [Flow Light](https://osu.ppy.sh/users/6705256) |
+| Commentators | ![][flag_CN] [hans1999](https://osu.ppy.sh/users/6679329), ![][flag_CN] [Flow Light](https://osu.ppy.sh/users/6705256), ![][flag_CN] [Rasis](https://osu.ppy.sh/users/1209363), ![][flag_CN] [644520858](https://osu.ppy.sh/users/5507069), ![][flag_CN] [wuhuanmin](https://osu.ppy.sh/users/935322) , ![][flag_CN] [704661982](https://osu.ppy.sh/users/7080633) |
+| Wiki | ![][flag_CN] [kj415j45](https://osu.ppy.sh/users/9367540) |
+
+## Links
+
+- [Livestream](https://live.bilibili.com/5333)
+- [Tournament QQ Group](https://jq.qq.com/?_wv=1027&k=5sTdKLp)
+
+## Mappool
+
+Coming Soon...
+
+## Ruleset
+
+### Overview
+
+1. This time, we don't have Qualification Stage or Group Stage. Everyone who qualified with the limitation can register then enter the Round of 16.
+2. This tournament is Double Elimination Tournament. This means that the winner moves to the next stage and the losing player gets moved to the Loser bracket.
+3. **Winning Condition**:
+    - In the Round of 16 and the Quarterfinals, player needs to win **5** maps to win a match. (Best-of-9).
+    - In the Semifinals and the Finals: Week 1, player needs to win **6** maps to win a match. (Best-of-11).
+    - In the Finals: Week 2, player needs to win **7** maps to win a match. (Best-of-13).
+4. Each stage will take one weekend.
+5. Every matches will be held on Saturday and Sunday of UTC+8.
+6. Scheduling will be handled by the Tournament Management. Schedules will be released on the Sunday before the first matches of the actual stage. If player has conflict with the scheduling, Tournament Management will try to amend the schedule so that it fairly accommodates for all the participant. However, no promises can be guaranteed:
+    - If player is in any other time zone, please notify the Management in advance.
+    - Apart from time zone reasons, the absent player(s) will be considered as forfeited, regardless of whether they have sent messages to the Management.
+
+### Tournament Rules
+
+1. **The 4th CJB osu!mania 4K Tournament _(CJB 4K 4th)_** is a one-on-one, osu!mania 4K specified Chinese regional tournament, hosted by CJB team.
```

Comment:
`**The 4th CJB osu!mania 4K Tournament _(CJB 4K 4th)_**, the fourth one-on-one osu!mania 4K specified Chinese regional tournament which is hosted by CJB team.` ->
`**This tournament is a one-on-one osu!mania 4K specified tournament.`

Only need to mention the specifics since you introduced it in this article already.